openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -84,7 +84,7 @@ set_url_and_curl(){
 
   URL="$BASE_URL/api/v1/monitoring/device/$UUID/?key=$KEY"
 
-  CURL_COMMAND="curl -s -w "%{http_code}""
+  CURL_COMMAND="curl -s -w "%{http_code}" --output "$TMP_DIR"/response.txt"
```

Comment:
I think the repetition of `"$TMP_DIR"/response.txt"` should be avoided

Reply:
:grimacing: Removed